impl/src/bin/cargo-raze.rs

```diff
@@ -76,16 +80,21 @@ fn main() -> Result<()> {
     .and_then(|d| d.deserialize())
     .unwrap_or_else(|e| e.exit());
 
+  // Load settings
   let settings = load_settings("Cargo.toml")?;
   if options.flag_verbose.unwrap_or(false) {
     println!("Loaded override settings: {:#?}", settings);
   }
 
-  let mut metadata_fetcher: Box<dyn MetadataFetcher> = match options.flag_cargo_bin_path {
-    Some(ref p) => Box::new(CargoMetadataFetcher::new(p, /*use_tempdir: */ true)),
+  // Fetch metadata
+  let metadata_fetcher: Box<dyn MetadataFetcher> = match options.flag_cargo_bin_path {
+    Some(ref p) => Box::new(CargoMetadataFetcher::new(
```

Comment:
Nit: I might be a bit famous for making variable names smaller (scope-declares-verbosity) but even by my standards I think this can be something like `path`

Reply:
Sorry, can you elaborate?

Reply:
So I _personally_ follow the practise of making my variable names as descriptive as possible without going to the point of utter verbosity.

Basically the closer a variable is in scope to where it will be used the shorter I feel I can get away with.

For example `x` / `y` is ok for say

```rust
some_iter.map(|x| x.thing)
```

whereas say for your code since `p` is a little further away from its use I might have mental forgotten what `p` relates to by the time it gets used, so in that case I would go with `path`

If it was, yet further away I might go longer and use `metadata_path` (and so on and so forth).

I will make exceptions if the context while short is just utterly obvious or ubiquitous e.g. https://github.com/simh/simh/blob/master/VAX/vax_cpu.c#L631 (I work on this :/) where sure `R` is very short but if you read any documentation on a vax its _the name_ for registers.

What I aim to avoid is either this https://github.com/simh/simh/blob/master/VAX/vax_cpu.c#L3792 (Great `i` is used everywhere and means a million little contextual things) or the other way https://chromium.googlesource.com/native_client/nacl-gcc/+/ng/master/libgo/go/net/fd_windows.go#36 (`hasLoadSetFileCompletionNotificationModes`)

In this context I suggest that `p` is a little too short as I found I had forgotten what `p` was by the time it was used.

Reply:
That was a good read. Thanks 😄 

I've updated the variable name.